docs: Update Delegation Policy (#724)
oasisprotocol/docs: docs/get-involved/delegation-policy.md

# Delegation Policy

Oasis Protocol Foundation (OPF) delegates ROSE tokens to node operators based on
their participation in the Oasis Mainnet, Testnet and ParaTimes on the network,
node reliability and performance, community engagement and overall support of
the network.

> :::danger

Reviewer: This should probably be "info" or at most "caution"?

> Starting in the first quarter of 2024, Oasis Protocol Foundation (OPF) will

Reviewer: ```suggestion
Starting in the first quarter of 2024, OPF will
```

initiate the Know Your Customer (KYC) process for all validators receiving

> delegation from OPF. It's essential for Node Operators to ensure that you have

Reviewer: ```suggestion
delegation from OPF. It is essential for node operators to ensure that you have
```

access to the email address provided in yours metadata submissions. This step
is crucial for completing the KYC procedure efficiently. Please update your
contact details if necessary.

:::

## Requirements for Receiving Delegations

### How to Earn Delegations?

> 1. Join the `#node-operators` channel on [Oasis Discord] server

Reviewer: ```suggestion
1. Join the `#node-operators` channel on [Oasis Discord] server.
```

> 2. Create your Testnet entity key and Mainnet [entity key]

Reviewer: ```suggestion
2. Create your Testnet entity key and Mainnet [entity key].
```
And similar below.

3. [Configure and start] Testnet consensus and paratime nodes
4. [Configure and start] Mainnet consensus node and paratime nodes that you are
able to  start
5. [Submit metadata] for Mainnet and Testnet

> 6. Apply for a delegation and pass KYC

Reviewer: Where to apply?

Author: I added a link to the form, but it would be nice if someone reviewed the form.

7. If selected - get onboarded on the delegation program
8. Actively participate on the Oasis Network Community server on Discord
   and help answer questions from other community members.

[Oasis Discord]: https://oasis.io/discord
[entity key]: ../node/run-your-node/validator-node#initialize-entity
[Configure and start]: ../node/run-your-node/validator-node#configuration

> [Configure and start]: ../node/run-your-node/validator-node#configuration

Reviewer: Duplicate links?

# Delegation Policy

The Oasis Protocol Foundation (OPF) delegates ROSE tokens to node operators who
run Oasis Consensus and ParaTime nodes on Mainnet and/or Testnet. Delegations
are based on node reliability, performance, community engagement, and overall
contributions to the network’s growth and stability.

## Requirements for Receiving Delegations

> To be eligible for a delegation, you have to join the network, follow the Code

Reviewer 2: Add links to the subsections below. E.g.

```
[join the network]: #join-the-network
```

of Conduct, and meet performance standards.

For guidance on how to increase your delegation over time, please refer to the
instructions at the bottom of this page.

### Join the Network

1. **Join the Community Channels**
   - Join the `#node-operators` channel on the [Oasis Discord server]
   - Or join the Telegram group: [@oasisnodeoperators]

2. **Create Your Entity Keys**
   - Generate your **Testnet** and **Mainnet** [entity keys]

3. **Set Up Your Nodes**
   - [Configure and start] your Testnet Consensus and ParaTime nodes
   - [Configure and start] your Mainnet Consensus and ParaTime nodes

4. **Submit Your Metadata**
   - [Provide metadata] for both **Mainnet** and **Testnet**

5. **Reach Out to the Node Operator Relationship Manager**
   - Message `@am3lody` on Discord or Telegram
   - Be prepared to answer any follow-up questions

6. **Stay Updated**
   - Follow network announcements:
      - 📢 [Oasis Discord server]: `#mainnet-announcements` and
      `#testnet-announcements`
      - 📢 [Oasis Mainnet Announcements]
      - 📢 [Oasis Testnet Announcements]

>       - 📢 [Oasis Node Operators Group]

Reviewer 2: This is a bit puzzling. I'd have two bullets, on for Discord and another for TG.

```
- Follow one of our social media channels for network announcements:
  - Discord: ...
  - Telegram: ...
```

Author: But telegram has one announcement channel for testnet and one for mainnet and one group. Added Discord and Telegram bullets.

7. **Engage with the Community**
   - Actively participate on the Oasis Network Community on Discord or Telegram
   - Follow the Code of Conduct and meet Performance Requirements
   - Help others and answer questions from fellow node operators and community
   members

[Oasis Discord server]: https://oasis.io/discord
[@oasisnodeoperators]: https://t.me/oasisnodeoperators
[entity keys]: ../node/run-your-node/validator-node#initialize-entity
[Configure and start]: ../node/run-your-node/validator-node#configuration
[Provide metadata]: ../node/run-your-node/validator-node#oasis-metadata-registry

> [Oasis Mainnet Announcements]: https://t.me/oasisMAINNETannouncements

Reviewer 2: These are TG links. Don't we use Discord for announcements?

Author: We use both now. Discord and Telegram.

of Conduct], and meet [performance requirements].

For guidance on how to increase your delegation over time, please refer to the
instructions at the bottom of this page.

### Join the Network

1. **Join the Community Channels**
   - Join the `#node-operators` channel on the [Oasis Discord server]
   - Or join the Telegram group: [@oasisnodeoperators]

2. **Create Your Entity Keys**
   - Generate your **Testnet** and **Mainnet** [entity keys]

3. **Set Up Your Nodes**
   - [Configure and start] your Testnet Consensus and ParaTime nodes
   - [Configure and start] your Mainnet Consensus and ParaTime nodes

4. **Submit Your Metadata**
   - [Provide metadata] for both **Mainnet** and **Testnet**

5. **Reach Out to the Node Operator Relationship Manager**
   - Message `@am3lody` on Discord or Telegram
   - Be prepared to answer any follow-up questions

6. **Stay Updated**
   - Follow network announcements:
      - Discord:
         - 📢 [Oasis Discord server]: `#mainnet-announcements` and
      `#testnet-announcements` 
      - Telegram:
         - 📢 [Oasis Mainnet Announcements]
         - 📢 [Oasis Testnet Announcements]

>          - 📢 [Oasis Node Operators Group]

Reviewer 2: Maybe squeeze into two lines only.

```suggestion
      - 📢 [Oasis Discord server]: [`#mainnet-announcements`], [`#testnet-announcements`], [`#node-operators`]
      - 📢 Telegram: [Oasis Mainnet Announcements], [Oasis Testnet Announcements] and [Oasis Node Operators Group]
      
[`#mainnet-announcements`]: https://discord.com/channels/748635004384313474/960599344745185330
[`#testnet-announcements`]: https://discord.com/channels/748635004384313474/967039075527827496
[`#node-operators`]: https://discord.com/channels/748635004384313474/748644177394532463
```

If there is any violation of the above Code of Conduct, Oasis Foundation
reserves the right to revoke your entire delegation.

[Oasis Metadata Registry]: https://github.com/oasisprotocol/metadata-registry

### Performance Requirements

1. Maintain active validator status.

> 2. Maintain >99% uptime for your validator node as well as Emerald and/or
>    confidential ParaTime nodes (if you run them).

Reviewer: ```suggestion
2. Maintain >99% uptime for your validator node as well as ParaTime nodes (if you run them).
```

issue that prevents your node from meeting the above Performance Requirements,
please reach out to the Oasis team as soon as possible.

## How to Increase Delegations?

> - Have a high uptime for Consensus, Emerald, Cipher, and Sapphire on both
> Testnet and Mainnet.

Reviewer: ```suggestion
- Have high uptime for Consensus, Sapphire and Cipher on both Testnet and Mainnet.
```

Testnet and Mainnet.
- Be prepared for planned and unplanned network upgrades.
- Vote on proposals.
- If you experience any problems or downtime, please communicate them with OPF
promptly so that we can take them into account.
- Have a meaningful amount of self-delegation and/or delegations from the Oasis
  community.
- Operate Emerald and/or Cipher ParaTime nodes on Mainnet.
- Operate validator and/or ParaTime nodes on Testnet
- Participate in community discussion on the network’s future roadmap.
- Actively participate on the Oasis Network Community server on Discord
  and help answer questions from other community members.
- Build or contribute to tools, services or dApps that benefit developers, node
  operators and/or the overall Oasis community.

> - Contribute code to Oasis projects.

Reviewer: Why has this been removed?
